creator-node/compose/env/base.env

```diff
@@ -42,15 +42,16 @@ spOwnerWalletIndex=
 
 
 # Sync / SnapbackSM configs
-snapbackReconfigEnabled=true
 snapbackJobInterval=10000 # ms
 snapbackModuloBase=2
 minimumDailySyncCount=5
 minimumRollingSyncCount=10
 minimumSuccessfulSyncCountPercentage=50
+
+snapbackHighestReconfigMode=ONE_SECONDARY
```

Comment:
lets not forget the corresponding k8s-manifests (stage + prod) change

Reply:
yup! will make PR soon

Reply:
made PR into k8s repo and tagged you!